duckduckgo/Android
# Update atb initialisation for returning users

Task/Issue URL: https://app.asana.com/0/488551667048375/1206059122944073/f

### Description
Implementation for adding variant `ru` for returning users

### Steps to test this PR

- Add a log in getVariantKey() implementation so you can check it's value

_New user install_
- Install from this branch
- Open logcat
- [x] Check defaultVariant is allocated (_emptyString_)

_Returning user install_
- Follow previous steps
- Perform a backup: Run `./test_cloud_backup.sh com.duckduckgo.mobile.android.debug` in the terminal
- Open the app
- [x] Check `ru` variant is allocated to the user

### No UI changes
